src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1030,18 +1203,72 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
+       if (laeroana_fv3cmaq) then
+         call gsi_fv3ncdf_read_v1(grd_fv3lam_tracerchem_ionouv,gsibundle_fv3lam_tracerchem_nouv,fv3filenamegin%tracers,fv3filenamegin)
+       endif
       endif
 
-      if( fv3sar_bg_opt == 0) then 
-        call GSI_BundleGetPointer ( gsibundle_fv3lam_dynvar_nouv, 'delp'  ,ges_delp ,istatus );ier=ier+istatus
+     if (laeroana_fv3cmaq) then
+        ier=0
+        call gsi_bundlegetpointer_fv3lam_tracerchem_nouv
+        if (ier/=0) call die(trim(myname),'cannot get pointers for fv3lam_tracerchem_nouv, ier =',ier)
+
+! Compute background amassi, amassj, amassk and pm2_5   
+       do k=1,nsig
+       do j=1,lon2
+          do i=1,lat2
+             ! imodes
+             ges_amassi(i,j,k)=ges_aso4i(i,j,k)+ges_ano3i(i,j,k)+ges_anh4i(i,j,k)+ &
+                               ges_anai(i,j,k) +ges_acli(i,j,k) +ges_aeci(i,j,k) + &
+                               ges_alvpo1i(i,j,k)+ges_asvpo1i(i,j,k)+ges_asvpo2i(i,j,k)+&
+                               ges_alvoo1i(i,j,k)+ges_alvoo2i(i,j,k)+ &
+                               ges_asvoo1i(i,j,k)+ges_asvoo2i(i,j,k)+ges_aothri(i,j,k)
+             ! jmodes
+             ges_amassj(i,j,k)=ges_aso4j(i,j,k)+ges_ano3j(i,j,k)+ges_anh4j(i,j,k)+ &
+                               ges_anaj(i,j,k) +ges_aclj(i,j,k) +ges_aecj(i,j,k) + &
+                               ges_alvpo1j(i,j,k)+ges_asvpo1j(i,j,k)+ges_asvpo2j(i,j,k)+&
+                               ges_asvpo3j(i,j,k)+ges_aivpo1j(i,j,k)+ &
+                               ges_axyl1j(i,j,k) +ges_axyl2j(i,j,k)+ges_axyl3j(i,j,k)+ges_atol1j(i,j,k)+ges_aothrj(i,j,k)+&
+                               ges_afej(i,j,k)   +ges_asij(i,j,k)  +ges_atij(i,j,k)  +ges_acaj(i,j,k)+&
+                               ges_amgj(i,j,k)   +ges_amnj(i,j,k)  +ges_aalj(i,j,k)  +ges_akj(i,j,k)+&
+                               ges_atol2j(i,j,k) +ges_atol3j(i,j,k)+&
+                               ges_abnz1j(i,j,k) +ges_abnz2j(i,j,k)+ges_abnz3j(i,j,k)+&
+                               ges_aiso1j(i,j,k) +ges_aiso2j(i,j,k)+ges_aiso3j(i,j,k)+&
+                               ges_atrp1j(i,j,k) +ges_atrp2j(i,j,k)+ges_asqtj(i,j,k)+&
+                               ges_aalk1j(i,j,k) +ges_aalk2j(i,j,k)+&
+                               ges_apah1j(i,j,k) +ges_apah2j(i,j,k)+ges_apah3j(i,j,k)+&
+                               ges_aorgcj(i,j,k) +ges_aolgbj(i,j,k)+ges_aolgaj(i,j,k)+&
+                               ges_alvoo1j(i,j,k)+ges_alvoo2j(i,j,k)+&
+                               ges_asvoo1j(i,j,k)+ges_asvoo2j(i,j,k)+ges_asvoo3j(i,j,k)+&
+                               ges_apcsoj(i,j,k)
+             ! kmodes
+             ges_amassk(i,j,k)=ges_aso4k(i,j,k)+ges_ano3k(i,j,k)+ges_anh4k(i,j,k)+ &
+                               ges_asoil(i,j,k)+ges_acors(i,j,k)+ges_aseacat(i,j,k)+ges_aclk(i,j,k)
+
+             ges_pm2_5(i,j,k)=ges_pm25at(i,j,k)*ges_amassi(i,j,k) +  ges_pm25ac(i,j,k)*ges_amassj(i,j,k) + ges_pm25co(i,j,k)*ges_amassk(i,j,k)
```

Comment:
This line exceeds the 132 character limit per line.  Please ensure that all lines meet the 132 character limit.